src/Application/Features/Employees/Handlers/CreateEmployeeHandler.cs

```diff
@@ -0,0 +1,29 @@
+﻿using MediatR;
+using MRA.AssetsManagement.Domain.Entities;
+
+public class CreateEmployeeCommand : IRequest<string>
```

Comment:
![image](https://github.com/srp-internships/MRA.AssetsManagement/assets/48281953/55a59e9f-5f2a-48f5-9a1e-d7e974583ff2)

Put queries to the `../Employees/Queries` folder
Put Commands to the `../Employees/Commands` folder

see other examples